.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: Create a report to help us improve the game
+labels: ["Type: Bug"]
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear and concise description of what the bug is. If you have a video of it happening, you may post it here.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: How To Reproduce
+    description: Steps to reproduce the behavior. If you have a video with the steps to recreate the bug, please post it here.
+    placeholder: |
+      1. Open ...
+      2. Do ...
+      3. See bug
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A clear and concise description of what is expected to happen.
+    placeholder: |
+      When I do X then Z should happen.
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Screenshots
+    description: If applicable, add screenshots to help explain your problem.
+  validations:
+    required: false
+
+- type: input
+  attributes:
+    label: Operating System
+    description: Windows version, Linux distro, etc.
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Renderer
+    options:
+      - DX9
+      - DX11
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: UI Framework
+    options:
+      - VGUI
+      - Panorama
+  validations:
+    required: true
```

Comment:
also maybe it'd be good to add a small explanation of how to tell which of these options you're using? i think some people get confused which is why they leave it blank, though this menu helped that already a good bit so i'm not 100% on it

![image](https://user-images.githubusercontent.com/62961532/130495433-029900d7-70a9-4680-9031-568361472fe5.png)

Reply:
maybe like `VGUI (default)`?

Reply:
👍